lib/hammer/utils.ex

```diff
@@ -15,6 +15,14 @@ defmodule Hammer.Utils do
     {stamp, key}
   end
 
+  def get_backend_module(which) do
```

Comment:
(Repeating a comment from above) I'm wondering if we could pass the `:single/which` param all the way down to `get_backend_module`, and have it do The Right Thing based on it's input.